DeepLinkSDK/DeepLink/DPLDeepLink.m

```diff
@@ -63,4 +65,60 @@ - (NSString *)description {
             [self.callbackURL description]];
 }
 
+
+#pragma mark - Equality
+
+- (BOOL)isEqual:(id)object {
+    if (self == object) {
+        return YES;
+    }
+    else if (![object isKindOfClass:[self class]]) {
+        return NO;
+    }
+    
+    return [self isEqualToDeepLink:object];
+}
+
+
+- (BOOL)isEqualToDeepLink:(DPLDeepLink *)deepLink {
```

Comment:
Passing a non-deeplink to `isEqualToDeepLink:` will currently crash. It may be worth doing a `class` check and just returning no here